priv/repo/migrations/20230105144400_create_node_stats_partitions_through_2030.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Ret.Repo.Migrations.CreateNodeStatsPartitionsThrough2030 do
+  use Ecto.Migration
+
+  import Ret.Repo.MigrationHelpers, only: [next_month: 2]
+
+  @max_year 2030
+
+  def up do
```

Comment:
Do we need to copy the rest of the table definition like the original migration? https://github.com/mozilla/reticulum/blob/0f94ced4785f1b24e2236d4c3fbe797d2e8e7f38/priv/repo/migrations/20180412225707_create_node_stats_table.exs#L7-L15

Reply:
Nope, the logical table already exists so all we have to do is create the partitions